metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -389,13 +389,21 @@ class MetalsLanguageServer(
           workspace,
           fingerprints
         )
+
+        val optJavaHome =
+          (userConfig.javaHome orElse JdkSources.defaultJavaHome)
+            .map(AbsolutePath(_))
+        val maybeJdkVersion: Option[JdkVersion] =
+          JdkVersion.maybeJdkVersionFromJavaHome(optJavaHome)
```

Comment:
If we already have javaHome thre is no need to run it.

Reply:
I just meant we should do `optJavaHome.getOrElse(JdkVersion.maybeJdkVersionFromJavaHome())`

Reply:
It's javaHome :facepalm: It's fine then :+1: